src/AppKit/NSBezierPath.cs

```diff
@@ -110,5 +110,15 @@ public unsafe void AppendPathWithGlyphs (uint[] glyphs, NSFont font)
 				_AppendPathWithGlyphs ((IntPtr)ptr, glyphs.Length, font);
 		}
 
+		public unsafe void AppendBezierPathWithCGGlyphs (uint[] glyphs, NSFont font)
```

Comment:
Also it's a new public API so it must be decorated with Mac (10,13) and others if applicable since the internal version won't be visible to developers